recipe/meta.yaml

```diff
@@ -22,9 +22,10 @@ requirements:
     - yarn <2
     - make
     - pkg-config
-    - sysroot_linux-64 ==2.17
+    - compilers
```

Comment:
Made this change in accordance to the latest recipe/meta.yml file in mainline: https://github.com/conda-forge/sagemaker-code-editor-feedstock/blob/main/recipe/meta.yaml